Sources/PackageDescription/LanguageStandardSettings.swift

```diff
@@ -10,66 +10,126 @@
 
 /// The supported C language standard to use for compiling C sources in the package.
 public enum CLanguageStandard: String, Encodable {
-    /// The identifier for the C89 language standard.
+
+    /// ISO C 1990.
     case c89
-    /// The identifier for the C90 language standard.
+
+    /// ISO C 1990.
     case c90
-    /// The identifier for the ISO 9899:1990 language standard.
-    case iso9899_1990 = "iso9899:1990"
-    /// The identifier for the ISO 9899:1994 language standard.
-    case iso9899_199409 = "iso9899:1994"
-    /// The identifier for the GNU89 language standard.
+
+    /// ISO C 1999.
+    case c99
+
+    /// ISO C 2011.
+    case c11
+
+    /// ISO C 2017.
+    @available(_PackageDescription, introduced: 999.0)
+    case c17
+
+    /// ISO C 2017.
+    @available(_PackageDescription, introduced: 999.0)
+    case c18
+
+    /// Working Draft for ISO C2x.
+    @available(_PackageDescription, introduced: 999.0)
+    case c2x
+
+    /// ISO C 1990 with GNU extensions.
     case gnu89
```

Comment:
Should this be "ISO C 1989"?  In general, do the years in the enum cases match the years of the ISO standards, or are there differences?